SeaPublicWebsite/Helpers/Global.cs

```diff
@@ -13,5 +13,7 @@ public static class Global
         public static string BasicAuthPassword => Config.GetAppSetting("BasicAuthPassword");
         public static string EpcAuthUsername => Config.GetAppSetting("EpcAuthUsername");
         public static string EpcAuthPassword => Config.GetAppSetting("EpcAuthPassword");
+        public static string GovUkNotifyApiKey => Config.GetAppSetting("GovUkNotifyApiKey");
+        public static string GovUkNotifyApiTestKey => Config.GetAppSetting("GovUkNotifyApiTestKey");

```

Comment:
This isn't used? If that's right, please remove it

Reply:
The Test key is not used, I'll remove that.
The regular Api key is, the api needs it